Microsoft.Toolkit.Uwp.SampleApp/Common/DelegateCommand.cs

```diff
@@ -21,7 +21,7 @@ namespace Microsoft.Toolkit.Uwp.SampleApp.Common
     /// </summary>
     public class DelegateCommand : ICommand
     {
-        private readonly Action commandExecuteAction;
+        private readonly Action<object> commandExecuteAction;
```

Comment:
Isn't there the DelegateCommand<T> we already have for passing the parameter scenario?

Reply:
Yes there is. I will revert this, makes no sense.